lib/cogctl/actions/groups/add.ex

```diff
@@ -1,61 +1,41 @@
 defmodule Cogctl.Actions.Groups.Add do
   use Cogctl.Action, "groups add"
+
   alias Cogctl.Actions.Groups
+  alias CogApi.Resources.User
+  alias CogApi.HTTP.Client
 
   def option_spec do
     [{:group, :undefined, :undefined, {:string, :undefined}, 'Group name (required)'},
-     {:user_to_add, :undefined, 'user', {:string, :undefined}, 'Username of user to add'},
-     {:group_to_add, :undefined, 'group', {:string, :undefined}, 'Name of group to add'}]
-  end
-
-  def run(options, _args, _config, endpoint) do
-    group = :proplists.get_value(:group, options)
-    user_to_add = :proplists.get_value(:user_to_add, options)
-    group_to_add = :proplists.get_value(:group_to_add, options)
-
-    with_authentication(endpoint,
-                        &do_add(&1, group, user_to_add, group_to_add))
-  end
-
-  defp do_add(_endpoint, :undefined, _user_to_add, _group_to_add) do
-    display_arguments_error
+     {:email, :undefined, 'email', {:string, :undefined}, 'User email address (required)'}]
   end
 
-  defp do_add(_endpoint, _group_name, :undefined, :undefined) do
-    display_arguments_error
+  def run(options, _args, _config, %{token: nil}=endpoint) do
+    with_authentication(endpoint, &run(options, nil, nil, &1))
   end
```

Comment:
It works the same as all of our other existing actions since `run` gets called with an endoint that has no token set, but I should remove the `%{token: nil}` from the function head in this once since I don't need to be that prescriptive.